privacyidea/static/components/config/views/config.html

```diff
@@ -61,6 +61,13 @@
                         <span translate>CAs</span>
                     </a>
                 </li>
+                <li ng-class="{active: $state.includes('config.periodictasks')}">
```

Comment:
Please move this menu entry after events. I think it is more logical there:

    policies, events, tasks, ....